src/geometries/plane.js

```diff
@@ -0,0 +1,13 @@
+var registerGeometry = require('../core/geometry').registerGeometry;
+var THREE = require('../lib/three');
+
+registerGeometry('plane', {
+  schema: {
+    height: {default: 1, min: 0},
+    width: {default: 1, min: 0}
+  },
+
+  init: function (data) {
+    this.geometry = new THREE.PlaneGeometry(data.width, data.height);
```

Comment:
Why are we dropping buffer geometries?

http://threejs.org/docs/#Reference/Core/BufferGeometry